### PR TITLE
goreleaser config and RPM+DEB packages

### PR DESCRIPTION
**Pull request by camzdiox**
goreleaser.yml and dist directory have been added. dist directory contains RPM and DEB packages.

### CODE REVIEW

File: goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    version_metadata: git
+    release: 1
+#    dependencies:
+#     - network-manager
+    description: |-
+      Geteduroam client for .deb based linux distributions.
+    overrides:
+     deb:
+      dependencies:
+       - network-managet
```
Comment (jwijenbergh) [inline goreleaser.yml:40]: typo?

**Comment by jwijenbergh** [comment]
Thanks a lot for this. Added a small comment

Could we not upload the dist folder here? I think only the goreleaser.yml file is fine and then we build it in the CI later

**Comment by mrvanes** [comment]
And could we have some documentation on how to actually use this configuration?

**Review by jwijenbergh — APPROVED** [review]
LGTM

**Comment by jwijenbergh** [comment]
Docs here: https://github.com/geteduroam/linux-app/wiki/Development-info

**Pull request merged**